contracts/contracts/coordination/subscription/BqETHSubscription.sol

```diff
@@ -113,7 +117,8 @@ contract BqETHSubscription is EncryptorSlotsSubscription, Initializable, Ownable
         );
         feeToken = _feeToken;
         adopter = _adopter;
-        baseFeeRate = _baseFeeRate;
+        initialBaseFeeRate = _initialBaseFeeRate;
+        baseFeeRateIncrease = _baseFeeRateIncrease;
```

Comment:
We need to validate that `_baseFeeRateIncrease` is a fraction of `INCREASE_BASE`.

Reply:
actually I thought that nothing prevents to have increase more than 100% so basically could be any number, but probably it's not great idea, yeah I can restrict it

Reply:
✔️